Source/Scene/ModelExperimental/buildDrawCommands.js

```diff
@@ -154,3 +170,112 @@ function deriveTranslucentCommand(command) {
   derivedCommand.renderState = RenderState.fromCache(rs);
   return derivedCommand;
 }
+
+function getIndexBuffer(primitiveRenderResources, debugWireframe, frameState) {
+  if (debugWireframe) {
+    return createWireframeIndexBuffer(primitiveRenderResources, frameState);
+  }
+
+  const indices = primitiveRenderResources.indices;
+  if (!defined(indices)) {
+    return undefined;
+  }
+
+  if (defined(indices.buffer)) {
+    return indices.buffer;
+  }
+
+  const typedArray = indices.typedArray;
+  const indexDatatype = IndexDatatype.fromSizeInBytes(
+    typedArray.BYTES_PER_ELEMENT
+  );
+
+  return Buffer.createIndexBuffer({
+    context: frameState.context,
+    typedArray: typedArray,
+    usage: BufferUsage.STATIC_DRAW,
+    indexDatatype: indexDatatype,
+  });
+}
+
+/**
+ * @private
+ */
+function createWireframeIndexBuffer(primitiveRenderResources, frameState) {
```

Comment:
Considering my other comment about handling `TRIANGLE_STRIP`/`TRIANGLE_FAN`, maybe we need a helper module for computing indices?